### PR TITLE
Add release notes for EDP 1.8.5

### PR DESCRIPTION
**Pull request by slega**
### For internal users - Please add a Jira DX PR ticket to the subject!
https://tyktech.atlassian.net/browse/DX-1244

<br>

---

### Preview Link

<br>

### Description
Release notes for EDP 1.8.5.
<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

___

## **Type**
documentation, enhancement

___

## **Description**
- Added detailed release notes for Tyk Enterprise Developer Portal v1.8.5, addressing the CVE-2024-3094 vulnerability.
- No breaking changes introduced in this release.
- Provided upgrade instructions for users on version 1.8.4 and older.
- Updated the documentation menu to include the new release notes.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>portal-1.8.5.md</strong><dd><code>Release Notes for Tyk Enterprise Developer Portal v1.8.5</code>&nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.5.md
<li>Added release notes for Tyk Enterprise Developer Portal v1.8.5.<br> <li> Documented the fix for CVE-2024-3094 vulnerability.<br> <li> Provided upgrade instructions and download links.

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4373/files#diff-11fa3cdaa590e39d1e1e65efacdd22a9b74b26d0ecd23208be145dc5d99c02b9">+48/-0</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>menu.yaml</strong><dd><code>Menu Update to Include v1.8.5 Release Notes</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/data/menu.yaml
- Updated the menu to include a link to the v1.8.5 release notes.

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4373/files#diff-a6789c66bfa0660f7fdbe4796002ab4777677bae59a0276689c840100ff0b0b2">+4/-0</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.5.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#### Future breaking changes
+This release doesn't introduce future breaking changes.
+
+#### Deprecations
+There are no deprecations in this release.
+
+#### Upgrade instructions
+If you are on 1.8.4 we advise you to upgrade ASAP directly to this release. This release doesn't introduce any changes to the theme, so theme upgrade is not required.
+
+If you are on 1.8.3 or older version please follow the [upgrade instructions]({{< ref "product-stack/tyk-enterprise-developer-portal/upgrading/theme-upgrades" >}}) to upgrade the portal's themes.
```
Comment (tbuchaillot) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.5.md:27]: @slega 1.8.4* ?

**Review by tbuchaillot — APPROVED** [review]
LGTM

**Review by letzya — APPROVED** [review]

**Pull request merged**